packages/cli/generation/ir-generator/src/__test__/__snapshots__/openapi/enum-casing.txt

```diff
@@ -974,11 +974,11 @@ exports[`generate IR from OpenAPI 1`] = `
                                           "map": [
                                             {
                                               "key": {
-                                                "jsonExample": "custom_fields",
+                                                "jsonExample": "key",
```

Comment:
similar issue here with `custom_fields` -> `key`

Reply:
This one's intentional, it's outlined at the end of the pr body, but since it was `{"custom_fields": "custom_fields"}` I updated to `{"key": "value"}`